### PR TITLE
[13.0][MIG] stock_return_request: Migration to v13.0

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa TT27479

ping @chienandalu @carlosdauden

### CODE REVIEW

**Review by chienandalu — APPROVED** [review]
Tested and code review :+1:

File: stock_return_request/models/stock_return_request.py [hunk 1]
```diff
@@ -0,0 +1,591 @@
...
+        selection=[
+            ("draft", "Open"),
+            ("confirmed", "Confirmed"),
+            ("done", "Done"),
+            ("cancel", "Cancelled"),
+        ],
+        default="draft",
+        readonly=True,
+        copy=False,
+        track_visibility="onchange",
```
Comment (victoralmau) [inline stock_return_request/models/stock_return_request.py]:
```suggestion
        tracking=True,
```

**Review by victoralmau — APPROVED** [review]

**Comment by sergio-teruel** [comment]
@victoralmau Changes done!!

**Pull request merged**